Resources/Prototypes/_Lust/Entities/Mobs/animals.yml

```
@@ -0,0 +1,57 @@
+    factions:
+    - Passive
+  - type: NPCRetaliation
+    attackMemoryLength: 10
+  - type: HTN
+    rootTask:
+      task: RuminantHostileCompound
+  - type: Tag
+    tags:
+    - VimPilot
+  - type: Carriable
+  - type: CanEscapeInventory
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)